adlfs/spec.py

```diff
@@ -1927,6 +1927,13 @@ async def _async_initiate_upload(self, **kwargs):
 
     _initiate_upload = sync_wrapper(_async_initiate_upload)
 
+    def get_chunks(self, data, chunk_size=1024**3):     #Keeping the chunk size as 1 GB
```

Comment:
```suggestion
    def _get_chunks(self, data, chunk_size=1024**3):     #Keeping the chunk size as 1 GB
```

Probably should keep this private until there's user demand for it. Will need to change where it's stored used below.